include/unicorn/x86.h

```diff
@@ -8,6 +8,15 @@
 extern "C" {
 #endif
 
+//Memory-Management Register fields (idtr, gdtr, ldtr, tr)
+//borrow from SegmentCache in qemu/target-i386/cpu.h
+typedef struct x86_mmr {
+    uint32_t selector;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
+    uint32_t limit;
+    uint32_t flags;
+} x86_mmr;
+
```

Comment:
No they are not segment registers in intel documents. They are Memory Management Registers (see volume 3, section 2.4)

CS, DS, SS, ES, FS and GS are segment registers.